frontend/src/components/vestigingscijfers/CitoScore.vue

```diff
@@ -34,18 +41,21 @@ export default {
   data () {
     return {
       'citoScoreData': null,
-      'citoScoreGem': null
+      'citoScoreGem': null,
+      'MISSING_DATA': false
     }
   },
   async mounted () {
     this.getData()
   },
   methods: {
     async getData () {
-      let score = await readPaginatedData(API_HOST + `/onderwijs/api/cito-score/?vestiging=${this.id}&jaar=2016`)
-      this.citoScoreData = score[0].cet_gem
-      let scoreGem = await getBbgaVariables(['OCITOSCH_GEM'], ['STAD'], [2016])
-      this.citoScoreGem = scoreGem[0].waarde
+      this.citoScoreData = _.get(await readPaginatedData(API_HOST + `/onderwijs/api/cito-score/?vestiging=${this.id}&jaar=2016`), '[0].cet_gem', null)
+      this.citoScoreGem = _.get(await getBbgaVariables(['OCITOSCH_GEM'], ['STAD'], [2016]), '[0].waarde', null)
```

Comment:
Is this really 2016?

Reply:
What happens in the future?

Reply:
I will tackle that problem, but not in this ticket which is about missing data and reporting about it. (OP-199)